generators/client/templates/angular/webpack/_webpack.prod.js

```diff
@@ -65,6 +65,19 @@ module.exports = webpackMerge(commonConfig({ env: ENV }), {
             ],
             exclude: ['node_modules/generator-jhipster']
         },
+        {
+            test: /\.ts$/,
```

Comment:
This was added to optimize the resource further. This loader removes the TS annotations and stuff which are not required for AOT. This loader produces a main bundle size of 388kb for below config, without this the size would be around 480 or something
```
{
  "generator-jhipster": {
    "promptValues": {
      "packageName": "com.mycompany.myapp",
      "nativeLanguage": "en"
    },
    "jhipsterVersion": "4.6.2",
    "baseName": "jh4test",
    "packageName": "com.mycompany.myapp",
    "packageFolder": "com/mycompany/myapp",
    "serverPort": "8080",
    "authenticationType": "jwt",
    "hibernateCache": "ehcache",
    "clusteredHttpSession": false,
    "websocket": "spring-websocket",
    "databaseType": "sql",
    "devDatabaseType": "h2Disk",
    "prodDatabaseType": "mysql",
    "searchEngine": "elasticsearch",
    "messageBroker": false,
    "serviceDiscoveryType": false,
    "buildTool": "maven",
    "enableSocialSignIn": true,
    "jwtSecretKey": "afc8cb1c9375cae7b67dac3ba1abc4599d9e2d35",
    "clientFramework": "angularX",
    "useSass": true,
    "clientPackageManager": "yarn",
    "applicationType": "monolith",
    "testFrameworks": [
      "protractor"
    ],
    "jhiPrefix": "jhi",
    "enableTranslation": true,
    "nativeLanguage": "en",
    "languages": [
      "en",
      "zh-tw",
      "fa",
      "ar-ly",
      "nl",
      "et",
      "fr",
      "hi"
    ]
  },
  "generator-jhipster-entity-audit": {
    "auditFramework": "custom"
  }
}
```